src/modules/airspeed_selector/airspeed_selector_params.c

```diff
@@ -194,3 +194,17 @@ PARAM_DEFINE_INT32(ASPD_FS_T2, 100);
  * @unit m/s
  */
 PARAM_DEFINE_FLOAT(ASPD_STALL, 10.0f);
+
+/**
+ * Maximal airspeed allowed for arming.
+ *
+ * It is not possible to arm the vehicle if the current airspeed estimate (equivalent airspeed) at arming is above this value.
+ * Airspeed on ground is either comming from wind or bad airspeed calibration.
+ *
+ * @group Airspeed Validator
+ * @unit m/s
+ * @min 0.0
```

Comment:
@sfuhrer Should we go for something more realistic here, like 0.5m/s? Zero would mean that your sensor needs to return exactly 0 airspeed all the time.

Reply:
yes set it to 0.5